client/src/index.js

```diff
@@ -1,14 +1,19 @@
-import React from "react";
+import React, {Suspense} from "react";
 import ReactDOM from "react-dom";
 import App from "pages/App";
 import reportWebVitals from "./utils/reportWebVitals";
 
 import "./index.css";
 
+// import i18n (needs to be bundled ;)) 
```

Comment:
remove comment?

Reply:
that came from the docs but ya oke i do it